### PR TITLE
Added core functionality

### PR DESCRIPTION
**Pull request by cynexit**
Added the base-frame needed for further development:
- Routing and parsing inside the webserver
- Base functionality for the scheduler (to be extended as soon as we know if "status" is feasible)
- Raw frame for Jobs to hook service- and engine betas into it 
- Fixed some RabbitMQ and config issues
- Added "dummy" actor as placeholder

### CODE REVIEW

File: src/main/scala/group/holmes/analytics/actors/Scheduler.scala [hunk 1]
```diff
@@ -1,19 +1,93 @@
...
+	// Get a list of all jobs
+	final case class List()
+
+	//Exposed objs
+	final case class ScheduledJob(ref: ActorRef, id: UUID, name: String, status: String)
 }
 
 class Scheduler(analyticEngineManager: ActorRef, analyticServiceManager: ActorRef) extends Actor with ActorLogging {
+	private val jobs = HashMap.empty[UUID, Scheduler.ScheduledJob]
+	implicit val timeout: Timeout = 10.seconds //TODO: Discuss sensible value / error mitigation strategy
```
Comment (webstergd) [inline src/main/scala/group/holmes/analytics/actors/Scheduler.scala:37]: maybe make this configurable? a separate pull request is fine by me.

**Review by webstergd — APPROVED** [review]
I wish I had more comments but this looks solid to me. I will leave it to ryan to do the pull and see if he can catch anything I missed.

File: src/main/scala/group/holmes/analytics/actors/Scheduler.scala (hunk 1 above)
Comment (cynexit) [reply src/main/scala/group/holmes/analytics/actors/Scheduler.scala:37]: Might be a good idea, so far I'm debating between a hard value and super-visioned restart vs. long timeout.

Comment (webstergd) [reply src/main/scala/group/holmes/analytics/actors/Scheduler.scala:37]: just let us know your thoughts. I am sure you will come up with something nice. Happy to help talk it out as well.

**Pull request merged**